### PR TITLE
fix: Adding a postman for the metadata retrieve for assets added by workflow api

### LINKED ISSUE #28338: Edit Contentlet: Workflow Action Endpoint does not return the `contentlet` metadata
### Parent Issue

https://github.com/dotCMS/core/issues/25445
### Task

We must retrieve the binary field `metadata` when executing a workflow action.

- **Endpoint:** `/api/v1/workflow/actions/${workflowActionId}/fire?inode=${contentletInode}&indexPolicy=WAIT_FOR`

#### Current Response

```
{
    "AUTO_ASSIGN_WORKFLOW": false,
    "__icon__": "jpgIcon",
    "archived": false,
    "baseType": "FILEASSET",
    "contentType": "Image",
    "underlyingFileName": "services-2.jpg",
    "url": "/images/pages/12345.jpg",
    "width": 844,
    "working": true
     ...Contentlet
}
```

### Proposed Objective

Core Features

### Proposed Priority

Priority 3 - Average

### Acceptance Criteria

The Workflow Action endpoint should retrieve both the binary field metadata in the `contentlet` information. Our current response is a `contentlet`. We only need to attach the metadata as we do in the Contentlet API.

#### Desire Response

```
{
    "AUTO_ASSIGN_WORKFLOW": false,
    "__icon__": "jpgIcon",
    "archived": false,
    "baseType": "FILEASSET",
    "contentType": "Image",
    "underlyingFileName": "services-2.jpg",
    "url": "/images/pages/12345.jpg",
    "width": 844,
    "working": true
     "Metadata": {
          ...binaryMetadata
     },
     ...Contentlet
}
```

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.

_No response_

### Assumptions & Initiation Needs

_No response_

### Quality Assurance Notes & Workarounds

_No response_

### Sub-Tasks & Estimates

_No response_

### PR DESCRIPTION
**Pull request by jdotcms**
Adding a postman to test the metadata retrieved as part of the response when the file asset is being added by the workflow api

This PR fixes: #28338

### CODE REVIEW

File: dotcms-postman/src/main/resources/postman/Workflow_Resource_Tests.json [hunk 1]
```diff
@@ -15600,12 +15600,15 @@
 									"    var jsonData = pm.response.json();",
 									"    pm.expect(jsonData.entity.fileAsset).to.include(\"greatings.txt\");",
 									"    pm.expect(jsonData.entity.contentType).to.be.eql(\"FileAsset\");",
+									"    pm.expect(jsonData.entity.metaData.name).to.be.eql(\"greatings.txt\");",
```
Comment (nollymar) [inline dotcms-postman/src/main/resources/postman/Workflow_Resource_Tests.json:15603]: "greetings"

File: dotcms-postman/src/main/resources/postman/Workflow_Resource_Tests.json [hunk 2]
```diff
@@ -15600,12 +15600,15 @@
...
+									"    pm.expect(jsonData.entity.metaData.title).to.be.eql(\"greatings.txt\");",
```
Comment (nollymar) [inline dotcms-postman/src/main/resources/postman/Workflow_Resource_Tests.json:15604]: "greetings"

Comment (nollymar) [reply dotcms-postman/src/main/resources/postman/Workflow_Resource_Tests.json:15604]: apparently the typo is everywhere on this file

File: dotcms-postman/src/main/resources/postman/Workflow_Resource_Tests.json (hunk 1 above)
Comment (jdotcms) [reply dotcms-postman/src/main/resources/postman/Workflow_Resource_Tests.json:15603]: jajaja true but it was like that, I am reusing a test

**Review by erickgonzalez — APPROVED** [review]
Besides the typo that Nolly mentions looks good

**Review by nollymar — APPROVED** [review]

**Pull request merged**